modules/nested-tabs/assets/js/frontend/utils.js

```diff
@@ -0,0 +1,7 @@
+export function isMenuInDropdownMode( elementSettings ) {
```

Comment:
@hein-obox This is duplicated code. You can rather move the mega-menu file somewhere more general and import it in both places.

Reply:
@louiswol94 

In my version there was a slight difference between the versions, but I have made them identical now.

Proposal: I can remove the version inside the MM in the pro version of this task.
In that way the base function is available in the Core plugin inside the Nested Tabs widget, and is inherited inside the MM widget in the pro version.

Reply:
@hein-obox Sorry I forgot this is in core. But okay cool, thanks.